### PR TITLE
SSCS-10865: Added arepanelmembersexcluded

### PR DESCRIPTION
**Pull request by nickhill111**
SSCS-10865

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by jarekPierchala — DISMISSED** [review]

File: benefit/data/sheets/EventToComplexTypes/EventToComplexTypes-nonprod.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "LiveFrom": "01/02/2023",
+    "FieldDisplayOrder": 1,
+    "PageID": "1.0"
+  },
+  {
+    "CaseEventID": "updateListingRequirements",
+    "CaseFieldID": "panelMemberExclusions",
+    "ListElementCode": "excludedPanelMembers",
+    "ID": "updateListingRequirementsExcludedPanelMembers",
+    "DisplayContext": "OPTIONAL",
```
Comment (amoslv1) [inline benefit/data/sheets/EventToComplexTypes/EventToComplexTypes-nonprod.json:17]: Shouldn't this be READONLY? OPTIONAL will add the text (Optional) to the element, which I don't think is what you want here?

**Review by amoslv1 — CHANGES_REQUESTED** [review]
Suggest changing DisplayContext on one element

File: benefit/data/sheets/EventToComplexTypes/EventToComplexTypes-nonprod.json (hunk 1 above)
Comment (nickhill111) [reply benefit/data/sheets/EventToComplexTypes/EventToComplexTypes-nonprod.json:17]: So in the update listing requirements, we want to allow the user to edit the list of excluded panel members but we don't want them to edit the panel
Scenario 2 - Existing exclusions

GIVEN that I need to update panel member exclusions

WHEN I select "Update listing requirements" and go to the screen

THEN display a new section "Panel Members Excluded" 

AND display any existing exclusions

AND allow the user to add new exclusions

AND allow the user to remove exclusions for panel members

Comment (amoslv1) [reply benefit/data/sheets/EventToComplexTypes/EventToComplexTypes-nonprod.json:17]: Oh right so the list of (excludded) panel members is actually writable. Ok

**Review by amoslv1 — DISMISSED** [review]

File: benefit/data/sheets/CaseEventToFields/CaseEventToFields.json [hunk 2]
```diff
@@ -375,7 +375,7 @@
   },
   {
     "CaseEventID": "adjournCase",
-    "CaseFieldID": "adjournCaseMedicallyQualifiedPanelMemberName",
+    "CaseFieldID": "adjournCasePanelMember2",
```
Comment (nickhill111) [inline benefit/data/sheets/CaseEventToFields/CaseEventToFields.json:378]: #############################
#############################
REVERT THIS BEFORE MERGING OR TESTING WITHOUT ADJOURNMENT FLAG
THIS IS FOR TESTING PURPOSES ONLY
#############################
#############################

File: benefit/data/sheets/CaseEventToFields/CaseEventToFields.json [hunk 3]
```diff
@@ -387,7 +387,7 @@
   },
   {
     "CaseEventID": "adjournCase",
-    "CaseFieldID": "adjournCaseOtherPanelMemberName",
+    "CaseFieldID": "adjournCasePanelMember3",
```
Comment (nickhill111) [inline benefit/data/sheets/CaseEventToFields/CaseEventToFields.json:390]: #############################
#############################
REVERT THIS BEFORE MERGING OR TESTING WITHOUT ADJOURNMENT FLAG
THIS IS FOR TESTING PURPOSES ONLY
#############################
#############################

**Review by AndreiTrushkov — APPROVED** [review]

**Review by paul-pearson — APPROVED** [review]

**Review by DGordonV1 — APPROVED** [review]

**Review by jarekPierchala — DISMISSED** [review]

**Review by AndreiTrushkov — APPROVED** [review]

**Review by jarekPierchala — APPROVED** [review]

**Pull request merged**